packages/webworker/lumberjack/src/lib/formatting/lumberjack-log-formatter-result.ts

```diff
@@ -1,6 +1,9 @@
 import { LumberjackLogPayload } from '../logs/lumberjack-log-payload';
 import { LumberjackLog } from '../logs/lumberjack.log';
 
+/**
+ * Represents what a LumberjackLogFormatter->formatLog function will return.
```

Comment:
suggestion: Rephrase
```suggestion
The return type of `LumberjackLogFormatter#formatLog`.
```